sdk/communication/Azure.Communication.CallAutomation/assets.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Any reason to remove this file?

Reply:
yes, the running live tests on test proxy are failing intermittently so that's why I have upload the actual tests recording for now.